### ISSUE TITLE
[bug] Cannot destructure property 'languages' of 'getRuntimeFromContext(...)' as it is undefined

### ISSUE THREAD

**Issue by WarningImHack3r** [inlang/inlang#1195]
### Problem

Since **sdk-js 0.8.0**, building a prerendered static SvelteKit app fails with this error:
```
TypeError: Cannot destructure property 'languages' of 'getRuntimeFromContext(...)' as it is undefined.
```
Then lower in the logs:
```
node:internal/event_target:1054
  process.nextTick(() => { throw err; });
                           ^
Error: 500 /
To suppress or handle this error, implement `handleHttpError` in https://kit.svelte.dev/docs/configuration#prerender
```

It works fine on 0.7.1 though.

### Expected behavior

To build properly

### Reproduction

1. Set up any SvelteKit project with the latest versions of everything
2. Make sure you `export const pretender = true;` (useful?)
3. Use `adapter-static` (useful?)
4. Build with `vite build`
5. Wait a bit and get this error

### `inlang.config.js` file

```JavaScript
/**
 * @type {import("@inlang/core/config").DefineConfig}
 */
export async function defineConfig(env) {
	const { default: pluginJson } = await env.$import(
		"https://cdn.jsdelivr.net/npm/@inlang/plugin-json@3/dist/index.js"
	);

	const { default: sdkPlugin } = await env.$import(
		"https://cdn.jsdelivr.net/npm/@inlang/sdk-js-plugin@0.7.1/dist/index.js"
	);

	const { default: standardLintRules } = await env.$import(
		"https://cdn.jsdelivr.net/npm/@inlang/plugin-standard-lint-rules@3/dist/index.js"
	);

	return {
		referenceLanguage: "en",
		plugins: [
			pluginJson({
				pathPattern: "./src/locales/{language}/*.json",
				variableReferencePattern: ["{", "}"]
			}),
			sdkPlugin({
				languageNegotiation: {
					strategies: [{ type: "localStorage" }]
				}
			}),
			standardLintRules()
		]
	};
}
```

### `npx @inlang/cli@latest config validate` file

```sh
ℹ 🔎 Validating the config file...                                                                                                                           21:08:03
✔ 🎉 Inlang config is valid!                                                                                                                                 21:08:03
```

### Other information

Browser: Arc 1.1.1
OS: macOS 13.5

**Comment by ivanhofer** [comment]
I can't reproduce the issue. Please share a repo with all the code we need to reproduce it.

**Comment by WarningImHack3r** [comment]
@ivanhofer you can use https://github.com/EmeraldHQ/Website as an example

**Comment by ivanhofer** [comment]
Thanks! I could find the issue and provided a fix in version `0.11.1`
